Assets/Scripts/SS3D/Systems/Chat/InGameChatWindow.cs

```diff
@@ -0,0 +1,233 @@
+using Coimbra;
+using FishNet;
+using SS3D.Core;
+using SS3D.Permissions;
+using SS3D.Systems.PlayerControl;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using TMPro;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+namespace SS3D.Engine.Chat
+{
+    /// <summary>
+    /// Behaviour responsible for handling chat UI in-game.
+    /// Should be attached to the in-game chat UI prefab.
+    /// It has tab, channel selector and can be dragged.
+    /// </summary>
+    public sealed class InGameChatWindow : ChatWindow, IDragHandler
+    {
+        [SerializeField] private bool defaultChat;
+        [SerializeField] private RectTransform tabRow = null;
+        [SerializeField] private ChatTab chatTabPrefab = null;
+        [SerializeField] private TMP_Dropdown channelDropDown = null;
+        [SerializeField] private CanvasGroup canvasGroup = null;
+
+        private readonly List<string> _channelDropdownOptions = new List<string>();
+        
+        private ChatTabData _currentTabData;
+
+        public CanvasGroup CanvasGroup => canvasGroup;
+
+        public RectTransform GetTabRow() => tabRow;
+
+        public int GetTabCount() => tabRow.childCount;
+        
+        protected override ChatChannel GetCurrentChatChannel() => 
+            chatChannels.GetChannels().FirstOrDefault(x => x.name == _channelDropdownOptions[channelDropDown.value]);
+        
+        public void Initialize()
+        {
+            if (defaultChat)
```

Comment:
defaultChat? defaultChat what? Setup default chats? use default chat? spawn default chat? Rename to be clearer. Also _lowerCase